crates/languages/src/deno.rs

```diff
@@ -70,7 +70,13 @@ impl LspAdapter for DenoLspAdapter {
         let os = match consts::OS {
             "macos" => "apple-darwin",
             "linux" => "unknown-linux-gnu",
-            "windows" => "pc-windows-msvc",
+            "windows" => {
+                if std::env::var("CARGO_CFG_TARGET_ENV").ok() == Some("msvc".to_string()) {
```

Comment:
`std::env::var("CARGO_CFG_TARGET_ENV")` is runtime value and possibly doesn't exist. Using `cfg!(target_env = "msvc")` is better.